README.MD

```diff
@@ -11,12 +11,14 @@ https://github.com/dlang/druntime/pull/2310
 
 STATUS:
 
-* almost builds on Windows - it just needs to fix the windows sdk reference
+* almost builds on Windows - just need to install Visual D on CI.  however Manu says tests pass on Windows for vector, string, optional
 
 TODO:
 
-1. implementation for various containers on Linux not yet done/finished
+0. fix Windows build and CI
+1. implementation for various containers on Linux not yet done/finished - gcc held back by internal struct pointer (awaiting opMove hook I think) and for clang the work just needs to be done.
 2. osx, freebsd impl and build not done at all I think
```

Comment:
vector is the only container that's not supported on all platforms, although cross-platform tests have not had a CI environment to prove they're big-free.